src/main/java/org/mockito/plugins/InstantiatorProvider.java

```diff
@@ -45,6 +45,7 @@
  *     <code>mockito-extensions/org.mockito.plugins.InstantiatorProvider</code>.
  * </p>
  *
+ * @deprecated Use {@link InstantiatorProvider2} instead
  * @since 21.10.15
```

Comment:
Bonus: if you can help us identify what version this was released in the past, that would be cool. It looks like this version is a date. It should be a number :)

Reply:
At the time the file was created version.properties showed 2.0.31-beta, so 2.0.31 ?